src/components/pwc-choices-2-option-bubble/pwc-choices-2-option-bubble.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Component,
+  h,
+  Prop,
+  EventEmitter,
+  Event,
+  Element
+} from "@stencil/core";
+import { PwcChoices2 } from "../../interfaces/PwcChoices2";
+
+@Component({
+  tag: "pwc-choices-2-option-bubble",
+  styleUrl: "../styles.scss",
+  shadow: true
+})
+export class PwcChoices2OptionBubbleComponent {
+  @Element() rootElement: HTMLPwcChoices2OptionBubbleElement;
+
+  @Prop() showCloseButton: boolean;
+  @Prop() option: PwcChoices2.IOption;
+  @Prop() indexInSelectedList: number;
+
+  @Event() closeClicked: EventEmitter<
+    PwcChoices2.IOptionBubbleCloseClickedEventPayload
+  >;
+
+  render() {
+    return (
+      <div class="bubble">
+        <span class="bubble-label">{this.option.label}</span>
+        {this.showCloseButton && (
+          <span
+            class="bubble-closeButton"
+            onClick={e => this.onCloseClicked(e)}
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
```

Comment:
Could be removed.

Reply:
fixed: ecb3acd67e76d92007a2759777423297891d240d